docs/command_reference.md

```diff
@@ -0,0 +1,235 @@
+<!--[metadata]>
++++
+title = "Notary Command Reference"
+description = "Notary command reference"
+keywords = ["docker, notary, command, notary command, reference"]
+[menu.main]
+parent="mn_notary"
+weight=99
++++
+<![end-metadata]-->
+
+# Notary Command Reference
+
+## Terminology Reference
+1. **GUN**: Notary uses Globally Unique Names (GUNs) to identify trusted collections.
+2. **Target**: Notary refers to the files the framework is used to distribute as "target files".  Target files are opaque to the framework. Whether target files are packages containing multiple files, single text files, or executable binaries is irrelevant to Notary.
+3. **Trusted Collection**: A trusted collection is a conceptual source of target files of interest to the application.
```

Comment:
I think I see what you mean by "source", but it may not be very clear since since notary doesn't necessarily identify the source of anything, and technically those files can be in disparate locations.  Maybe just "a list of target files"?  :|  I don't have a good suggestion.